hibernate-core/src/test/java/org/hibernate/orm/test/id/uuid/interpretation/UUIDBasedIdInterpretationTest.java

```diff
@@ -90,6 +92,7 @@ public void testMySQL() {
 
 	@Test
 	@TestForIssue( jiraKey = "HHH-10564")
+	@RequiresDialect( PostgreSQL94Dialect.class )
```

Comment:
In Hibernate 6 we don't use the version specific dialect sub-classes anymore but configure the dialect during construction instead, so please use:

```suggestion
	@RequiresDialect( PostgreSQLDialect.class )
```